lib/utils.js

```diff
@@ -205,6 +205,27 @@ utils.getResourceUID = function getResourceUID(namespace, resource) {
         .update(namespace + resource, 'binary').digest('hex');
 };
 
+/**
+ * Validate request headers to determine if readable characters
+ * @param {object} headers - headers attached to the http request (lowercased)
+ * @return {boolean} - returns true if headers are readable, false if not
+ */
+utils.checkReadableHeaders = function checkReadableHeaders(headers) {
```

Comment:
`every` tests whether all elements in the array pass the test implemented by the provided function.
I get my array from `Object.keys(headers).filter(h => !h.startsWith('x-amz-meta-'))` and my test checks that each character is readable.

Is it clearer? :)   
Thanks for the review.